recipes/flann/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "1.8.4" %}
+
+package:
+    name: flann
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/mariusmuja/flann.git
+    git_tag: 40d63fc7db5d1f6dd67364f24e4579aedb57e7b6
+
+build:
+    number: 0
+    string: {{PKG_BUILDNUM}}_g{{GIT_FULL_HASH[:7]}}
```

Comment:
This is the most recent release. I added this string because someone along the way recommended it. I'm happy to change this if you'd prefer